fire/cli/niv/_ilæg_nye_punkter.py

```diff
@@ -98,7 +98,7 @@ def ilæg_nye_punkter(projektnavn: str, sagsbehandler: str, **kwargs) -> None:
 
     # Opret punkter
     fire.cli.print(f"Behandler punkter")
-    for i in range(n):
+    for i in nyetablerede.index:
```

Comment:
Elegant ændring, når man véd, at Pandas ignorerer tomme rækker. For mig ville det være rart, hvis rationalet kom med som en kort kommentar.

```python
# Index frasorterer automatisk blanke linjer og er dermed mere robust.
# Bemærkes, at felter med usynlige tegn som mellemrum IKKE resulterer
# i en tom linje og en sådan vil fremtvinge en fejl pga ugyldigt input.
for i in nyetablerede.index:
   ...
```

Reply:
God pointe, der er taget hånd om.